datasets/senti_ws/senti_ws.py

```diff
@@ -0,0 +1,133 @@
+        ),
+    ]
+
+    DEFAULT_CONFIG_NAME = "pos-tagging"
+
+    def _info(self):
+
+        if (
+            self.config.name == "pos-tagging"
+        ):  # the pos-tags are ["NN", "VVINF", "ADJX", "ADV"] -> ["noun", "verb", "adjective", "adverb"]
+            features = datasets.Features(
+                {
+                    "word": datasets.Value("string"),
+                    "pos-tag": datasets.Value("string"),
+                }
+            )
+        else:  # This is an example to show how to have different features for "first_domain" and "second_domain"
+            features = datasets.Features(
+                {
+                    "word": datasets.Value("string"),
+                    "sentiment-score": datasets.Value("float32"),
+                }
+            )
+        return datasets.DatasetInfo(
+            # This is the description that will appear on the datasets page.
+            description=_DESCRIPTION,
+            # This defines the different columns of the dataset and their types
+            features=features,  # Here we define them above because they are different between the two configurations
+            # If there's a common (input, target) tuple from the features,
+            # specify them here. They'll be used if as_supervised=True in
+            # builder.as_dataset.
+            supervised_keys=None,
+            # Homepage of the dataset for documentation
+            homepage=_HOMEPAGE,
+            # License for the dataset if available
+            license=_LICENSE,
+            # Citation for the dataset
+            citation=_CITATION,
+        )
+
+    def _split_generators(self, dl_manager):
+        """Returns SplitGenerators."""
+        # TODO: This method is tasked with downloading/extracting the data and defining the splits depending on the configuration
+        # If several configurations are possible (listed in BUILDER_CONFIGS), the configuration selected by the user is in self.config.name
+
+        # dl_manager is a datasets.download.DownloadManager that can be used to download and extract URLs
+        # It can accept any type or nested list/dict and will give back the same structure with the url replaced with path to local files.
+        # By default the archives will be extracted and a path to a cached folder where they are extracted is returned instead of the archive
+        my_urls = _URLs
+        data_dir = dl_manager.download_and_extract(my_urls)
+        return [
+            datasets.SplitGenerator(
+                name=datasets.Split.TRAIN,
+                # These kwargs will be passed to _generate_examples
+                gen_kwargs={
+                    "sourcefiles": sorted(
+                        os.path.join(data_dir[0], f)
+                        for f in ["SentiWS_v2.0_Positive.txt", "SentiWS_v2.0_Negative.txt"]
+                    ),
```

Comment:
```suggestion
                    "sourcefiles": [
                        os.path.join(data_dir[0], f)
                        for f in ["SentiWS_v2.0_Positive.txt", "SentiWS_v2.0_Negative.txt"]
                    ],
```
you don't need to sort here